src/reducers/listReducer.js

```diff
@@ -25,6 +25,15 @@ const add = (list, item, comparator = null) => {
   return list && !_.find(list, test) ? [...list, item] : list
 }
 
+const addAll = (list, items, comparator = null) => {
+  let newList = list
+  items.forEach(item => {
+    const test = comparator ? element => comparator(element, item) : element => element === item
+    newList = newList && !_.find(newList, test) ? [...newList, item] : newList
```

Comment:
Hmmm, likely should use a lodash union-ish function. this iwll make 00s of copies of the array (`[...newList]` makes a copy of the array).